templates/setup.py.jj2

```diff
@@ -4,6 +4,7 @@
 {% block shebang %}
 #!/usr/bin/env python3
 {% endblock %}
+{% set min_ver_python = min_ver_python|default([2,6]) %}
```

Comment:
Please add space after a comma. This is so that we follow PEP8. The same goes for all the changes you made below.